drivers/linstorvolumemanager.py

```diff
@@ -841,31 +841,33 @@ def resize_volume(self, volume_uuid, new_size):
         self.ensure_volume_is_not_locked(volume_uuid)
         new_size = self.round_up_volume_size(new_size) // 1024
 
-        retry_count = 30
-        while True:
-            result = self._linstor.volume_dfn_modify(
-                rsc_name=volume_name,
-                volume_nr=0,
-                size=new_size
-            )
-
-            self._mark_resource_cache_as_dirty()
-
-            error_str = self._get_error_str(result)
-            if not error_str:
-                break
+        # We can't resize anything until DRBD is up to date
+        # We wait here for 5min max and raise an easy to understand error for the user
+        # 5min is an arbitrary time, it's impossible to get a fit all situation value
+        # and it's currently impossible to know how much time we have to wait
+        # This is mostly an issue for thick provisioning, thin isn't affected
```

Comment:
I think we'll need to discuss comment formatting for the new stack, especially regarding period at the end of sentences. I'm a fan of this for pydocstyle, but also for comments. :)